### PR TITLE
Add support for multiple outputs with the same loss func in keras

### LINKED ISSUE #10: Handling more than one output, but only one loss
I have a model that has three outputs (named "usage", "clients", and "sessions") and one loss function (mean absolute percentage error.) As shown in the picture below, the names of my plots are a little messed up. I believe the upper right plot should be titled "usage_loss".
image

I think the problem is in this section of the code, but I'm not sure how to fix it.

https://github.com/stared/livelossplot/blob/fd72c53bd82d0c0052478c735edd356892f6f7f6/livelossplot/keras_plot.py#L54-L59

### PR DESCRIPTION
**Pull request by jonahweissman**
I confirmed that this code works for both my model and `keras_example.ipynb`. I haven't confirmed that it works with multiple outputs and multiple loss functions, however.

If there are any code-style changes you want me to make, just let me know.

closes #10

### CODE REVIEW

**Comment by stared** [comment]
Looks good to me, thanks! :)
